### PR TITLE
update/cleanup tests

### PR DESCRIPTION
**Pull request by jchip**

### CODE REVIEW

**Comment by datvong-wm** [comment]
what's wrong with async?

**Comment by jchip** [comment]
@datvong-wm nothing, it's not needed unless `await` is used in the function.

**Comment by datvong-wm** [comment]
await was used in the function

**Comment by jchip** [comment]
please comment at the line of code that has the issue

File: test/spec/electrode.spec.js [hunk 1]
```diff
@@ -92,108 +92,102 @@ describe("electrode-server", function() {
...
+        expect(error).to.be.an("Error");
+        expect(error.message).includes("test listner error");
+      }
+    );
   });
 
-  it("should fail for listener errors from decor array with func", async function() {
-    let error;
-    try {
-      await electrodeServer({}, [require("../decor/decor4")]);
```
Comment (datvong-wm) [inline test/spec/electrode.spec.js]: this seems more readable than using asyncVerify

Comment (jchip) [reply test/spec/electrode.spec.js]: ok, that's debatable, but readability is not the reason asyncVerify's created or used.

**Pull request merged**